tests/testthat/test-set-credentials.R

```diff
@@ -0,0 +1,55 @@
+# Run this test for users to make sure the set credentials from R object features
+# of doAzureParallel are still working
+context("set credentials from R object scenario test")
```

Comment:
Is the 'old' way of passing a file still working? Probably worth adding tests to make sure those calls work as before.